http_logs/test_procedures/default.json

```diff
@@ -831,8 +831,8 @@
         {
           "name": "match-all",
           "operation": "default",
-          "warmup-iterations": 500,
-          "iterations": 100,
+          "warmup-iterations": {{ default_warmup_iterations or warmup_iterations | default(500) | tojson }},
```

Comment:
Some of these are using `default_warmup_iterations` and `default_iterations`. Could we update them to match the operation name?